client/webserver/site/src/html/settings.tmpl

```diff
@@ -35,7 +35,16 @@
       <div id="exchanges" {{if eq (len .UserInfo.Exchanges) 0}} class="d-hide"{{end}}>
         <h5>[[[registered dexes]]]</h5>
         {{range $host, $xc := .UserInfo.Exchanges}}
-          <a href="/dexsettings/{{$host}}"><button class="bg2 selected"><div class=text-break>{{$host}}<span class="dex-settings-icon ico-settings"></span></div></button></a>
+          <a href="/dexsettings/{{$host}}">
+            <button class="bg2 selected">
+              <div class=text-break>
+                <img class="mini-icon m-1" src="/dexlogo/{{$host}}">
```

Comment:
Logo is a little big relative to the text size. 
![image](https://user-images.githubusercontent.com/6109680/214315113-00c62e67-4ccb-4fca-9efe-dbc351c1b2d6.png)
Could increase text size or decrease logo size or both.

Reply:
I intentionally made the logo bigger so it's more visible but seems it should match the text size.